Tests/SnapshotTests.swift

```diff
@@ -155,10 +162,9 @@ class SnapshotTests: XCTestCase {
           _experimentalFeatureConfiguration: ExperimentalFeatureConfiguration(
             useNewRenderingEngine: usingExperimentalRenderingEngine))
 
-        // Set up the animation view with a valid frame and layout
+        // Set up the animation view with a valid frame
         // so the geometry is correct when setting up the `CAAnimation`s
         animationView.frame.size = animation.snapshotSize
-        animationView.layoutIfNeeded()
```

Comment:
When I was testing things I noticed the tests pass without this, so I figured we may as well remove it